src/test/java/org/datadog/jmxfetch/TestApp.java

```diff
@@ -100,6 +100,7 @@ public void testRegexpAliasing() throws Exception {
 
         assertMetric("this.is.100.bar.baz", tags, 4);
         assertMetric("org.datadog.jmxfetch.test.baz.hashmap.thisis0", tags, 4);
+        assertMetric("this.is.thousand.1000.0", 1000, tags, 4);
```

Comment:
what's the actual value of the metric when the JMX attribute's value is a string? Could you add a test on that? (you can add attributes to the test app that exposes JMX attributes)

AFAIK this feature is only really useful if the actual value is a string actually